opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "1.2"
+name: "leveldb"
+version: "1.1.0"
+maintainer: "mfp@acm.org"
+homepage: "https://github.com/mfp/ocaml-leveldb"
+license: "LGPL+static"
+doc: ["https://github.com/mfp/ocaml-leveldb/blob/master/README.md"]
+build: [
+  ["omake" "-j9"]
+  ["omake" "install" "prefix=%{prefix}%"]
+]
+build-test: [["omake" "test"] {"%{ounit:installed}"}]
+remove: [
+  ["ocamlfind" "remove" "leveldb"]
+]
+depends: [
+  "ocamlfind" {build}
+  "omake"
+  "conf-leveldb"
+  "ounit"
```

Comment:
should `ounit` be marked as `{test}`? Also you can probably mark `{omake}` as `{build}` as well.